app/tests/algorithms_tests/test_views.py

```diff
@@ -1137,3 +1138,66 @@ def test_job_time_limit(client):
 
     assert job.algorithm_image == algorithm_image
     assert job.time_limit == 600
+
+
+@pytest.mark.django_db
+def test_evaluations_are_filtered(client):
```

Comment:
Shouldn't this test also check that the ranking is taken into considertaion correctly?